ArduCopter/ArduCopter.cpp

```diff
@@ -111,6 +111,7 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
     SCHED_TASK(three_hz_loop,          3,     75),
     SCHED_TASK(compass_accumulate,   100,    100),
     SCHED_TASK(barometer_accumulate,  50,     90),
+    SCHED_TASK(barometer_adjust,     400,     50),
```

Comment:
The comment in L80-L93 says "400 = 1Hz". But the macro definition says rate_hz. 

So what is correct? May we should fix either the comment or the macro?